.vib/owncloud/cypress/cypress/support/commands.js

```diff
@@ -0,0 +1,35 @@
+const COMMAND_DELAY = 800;
+const BASE_URL = 'http://vmware-owncloud.my/';
+
+for (const command of ['click']) {
+  Cypress.Commands.overwrite(command, (originalFn, ...args) => {
+    const origVal = originalFn(...args);
+
+    return new Promise((resolve) => {
+      setTimeout(() => {
+        resolve(origVal);
+      }, COMMAND_DELAY);
+    });
+  });
+}
+
+Cypress.Commands.overwrite('visit', (originalFn, url, options) => {
+  return originalFn(`${BASE_URL}${url}`, options);
+});
+
+Cypress.Commands.add(
+  'login',
+  (username = Cypress.env('username'), password = Cypress.env('password')) => {
+    cy.clearCookies();
```

Comment:
Unless explicitly needed to, I recall it was safe to delete this line as Cypress would indeed clear the cookies before every test run.